status-im/status-react
# Emoji reactions doesn't resend when sent immediately after network disconnect

fixes #11553

### Summary

Switching to `status-go` version that fixes problem with incorrect "sent" signal. After that emojis that sent immediately after network disconnect won't be lost.

#### Platforms
- Android
- iOS

#### Areas that maybe impacted
Everything related to message delivering and confirmation

##### Functional

- 1-1 chats
- public chats
- group chats


### Steps to test


- Open Status
- turn off network connection and quickly add emoji reaction to any message
- at this moment reaction can't be seen on other device
- turn on network connection
- make sure reaction seen on other device after short period of time

status: ready
